www/ns.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
This file can be deleted after you fix up the googletagmanager link properly above.